app/renderer/src/components/Inspector/Inspector.jsx

```diff
@@ -314,7 +316,14 @@ const Inspector = (props) => {
                         </span>
                       }
                       extra={
-                        <span>
+                        <Row>
+                          <Input
+                            size="middle"
+                            allowClear
+                            placeholder="Search in page source"
```

Comment:
Please add this to the translations file

Reply:
Moved the string to translations file.